<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frms_selecionaveiculo</class>
 <widget class="QDialog" name="frms_selecionaveiculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seleção de Marca e Modelo</string>
  </property>
  <widget class="QTableWidget" name="tw_selecionaVeiculo">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>63</y>
     <width>590</width>
     <height>275</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>31</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="txt_filtrarVeiculo"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_filtrarVeiculo">
      <property name="text">
       <string>Filtrar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>8</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Filtrar:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cb_filtrar"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_confirmarVeiculo">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirmar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
